src/middleware/editProfile.js

```diff
@@ -0,0 +1,17 @@
+import model from '../sequelize/models';
+
+const { User } = model;
+
+const canEditProfile = async (req, res, next) => {
+  const suggestedUsername = req.params.username;
+  const userModel = await User.findOne({ where: { email: req.userData.email } });
+  const trueUsername = userModel.userName;
+
+  if (trueUsername !== suggestedUsername) {
+    return res.status(403).send({
+      error: 'sorry! you can not edit the profile that is not yours'
```

Comment:
For this error message. I think it'd be better if the message started with a capital letter and instead of `can not`, `cannot` would be a better option in this case [Reference to this article](https://www.grammarly.com/blog/cannot-or-can-not/)

Reply:
on it. thanks